api/internal/conf/conf.go

```diff
@@ -54,6 +54,7 @@ var (
 	SSLDefaultStatus = 1 //enable ssl by default
 	ImportSizeLimit  = 10 * 1024 * 1024
 	PIDPath          = "/tmp/manager-api.pid"
+	FilePathSet		 = "/conf/conf.yaml"
```

Comment:
Please take care of code style.

Reply:
@vinayaksh42 
You could run `go fmt` to format the code.

Reply:
okay i will do that and make another commit to this pr